### PR TITLE
Keep a record of the works done so far as a pre-release v.0.1.0

### PR DESCRIPTION
**Pull request by Farhad-Shabani**
## Summary

Pre-release the `basecoin-rs` to record the work done so far and to serve as a reference for performing tests with specific versions of `IBC-rs` and `Tendermint-rs` 

## Remarks
- Some fixations applied to pass CI
- Versioning restarted to 0.1.0-alpha (as initial), but if continue from prev 0.2.0 would be better, please let me know

### CODE REVIEW

**Review by hu55a1n1 — APPROVED** [review]
Thanks, @Farhad-Shabani! Added one suggestion about starting with `v0.1.0`, but otherwise looks good to me! :raised_hands: 
Also, I wonder if users would find it helpful if the crate was reorganized as a lib with a `cli` bin. (Can discuss this later.)

> Versioning restarted to 0.1.0-alpha (as initial), but if continue from prev 0.2.0 would be better, please let me know

This should be fine as we don't have a public release yet.

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tendermint-basecoin"
-version = "0.2.0"
+version = "0.1.0-alpha"
```
[the remark this answers is not in the document]
Comment (Farhad-Shabani) [reply Cargo.toml]: [3a47ab3](https://github.com/informalsystems/basecoin-rs/pull/69/commits/3a47ab3611026cd92f0b40dd1acb0c026595779f)

[the remark this answers is not in the document]
Comment (hu55a1n1) [reply Cargo.toml]: We also need to change the version in the changelog. :)

[the remark this answers is not in the document]
Comment (Farhad-Shabani) [reply Cargo.toml]: ahh, thank you :grimacing:

**Pull request merged**